src/multiversioning.zig

```diff
@@ -1063,7 +1063,7 @@ pub const Multiversion = struct {
     fn handle_error(self: *Multiversion, result: anyerror) void {
         assert(self.stage != .init);
 
-        log.err("binary does not contain valid multiversion data: {}", .{result});
+        log.warn("binary does not contain valid multiversion data: {}", .{result});
```

Comment:
I'd maybe leave both of these as errors --- during upgrades, it feels better to ping proactively